### PR TITLE
added rules content

### PR DESCRIPTION
**Pull request by Kaz-ookid**
added rules content :D

### CODE REVIEW

File: app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+class RulesStrings {
+    companion object {
+
+        const val RULES_BASICS_TITLE = "Basics"
+        const val RULES_BASICS_TEXT =
+            "Before starting a game, the game owner must choose the following parameters:\n\n" +
+                    "- The duration of each round (from minutes to weeks!)\n\n" +
+                    "- The number of players (from 2 up to 8) \n\n" +
+                    "- The game mode : Richest player, Last standing and Landlord!\n\n" +
+                    "- The number of rounds\n\n" +
```
Comment (vrooomer) [inline app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt:12]: Is it logic to have a fixed number of rounds for "Last Standing" mode?

_edit: you should somehow precise that this number isn't useful for this gamemode_

File: app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+        const val RULES_RICHEST_PLAYER_TITLE = "Richest player mode"
+        const val RULES_RICHEST_PLAYER_TEXT =
+            "In this mode, the winner is the player that end up with the highest balance when the game ends " +
+                    "(when all the rounds are played). Build a powerful empire and be the RICHEST!\n\n"
+
+        const val RULES_LAST_STANDING_TITLE = "Last standing mode"
+        const val RULES_LAST_STANDING_TEXT =
+            "In this mode, the winner is the player that is the last in the game. " +
+                    "This mode is not round base, this means it runes until someone wins! " +
```
Comment (vrooomer) [inline app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt:27]: runs*

File: app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            "In this mode, the main way of getting money is by passively collecting rent from your buildings. " +
+                    "Player will be taxed whenever they step in a building owned by someone else! " +
+                    "In this mode, you can randomly give starting buildings to players, to spice things up! " +
+                    "The winner is the player that has the highest balance when the game ends."
+
+        const val RULES_DURING_ROUND_TITLE = "During a round"
+        const val RULES_DURING_ROUND_TEXT =
+            "During rounds, you will not see other players moves neither their actions. Once the round ends, players will be able to see the moves of other players and the results of their actions!\n" +
+                    "At the beginning of each rounds, players will get to roll a dice and get prompted 3 locations to visit. " +
+                    "They will have to chose only one, which they will have to physically go to by the end of the round, otherwise you will get a malus!\n\n" +
```
Comment (vrooomer) [inline app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt:41]: So we agree on fixing the number of chosen locations to one?

Comment (vrooomer) [inline app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt:41]: choose*

**Review by vrooomer — APPROVED** [review]
Nothing to say as there is no code here, but nice job for the rules!! Even though having "empty empty empty" was funny :P

Approved but please check the review I made :)

~ v:)

File: app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt (hunk 1 above)
Comment (Kaz-ookid) [reply app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt:12]: it is said in the landlord mode description

**Pull request merged**